classes/task/emit_task.php

```diff
@@ -50,10 +50,13 @@ public function execute() {
         $store = new store($manager);
 
         $events = $DB->get_records('logstore_xapi_log');
-        $store->process_events($events);
-
-        $DB->delete_records_list('logstore_xapi_log', 'id', array_keys($events));
-
+        $store_return = $store->process_events($events);
+		foreach(array_keys($store_return) as $event_id)
+			{
+			if ($store_return[$event_id] == 'success')
```

Comment:
I made some tests with simple events, especially the course completed and everything went fine. Can you show me some Moodle events that produce multiple xAPI events so I can test them?